nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/reporting/AtlasNiFiFlowLineage.java

```diff
@@ -0,0 +1,714 @@
+            .required(false)
+            .expressionLanguageSupported(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor ATLAS_PASSWORD = new PropertyDescriptor.Builder()
+            .name("atlas-password")
+            .displayName("Atlas Password")
+            .description("Password to communicate with Atlas.")
+            .required(false)
+            .sensitive(true)
+            .expressionLanguageSupported(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .build();
+
+    static final PropertyDescriptor ATLAS_CONF_DIR = new PropertyDescriptor.Builder()
+            .name("atlas-conf-dir")
+            .displayName("Atlas Configuration Directory")
+            .description("Directory path that contains 'atlas-application.properties' file." +
+                    " If not specified and 'Create Atlas Configuration File' is disabled," +
+                    " then, 'atlas-application.properties' file under root classpath is used.")
+            .required(false)
+            .expressionLanguageSupported(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor ATLAS_NIFI_URL = new PropertyDescriptor.Builder()
+            .name("atlas-nifi-url")
+            .displayName("NiFi URL for Atlas")
+            .description("NiFi URL is used in Atlas to represent this NiFi cluster (or standalone instance)." +
+                    " It is recommended to use one that can be accessible remotely instead of using 'localhost'.")
+            .required(true)
+            .expressionLanguageSupported(true)
+            .addValidator(StandardValidators.URL_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor ATLAS_DEFAULT_CLUSTER_NAME = new PropertyDescriptor.Builder()
+            .name("atlas-default-cluster-name")
+            .displayName("Atlas Default Cluster Name")
+            .description("Cluster name for Atlas entities reported by this ReportingTask." +
+                    " If not specified, 'atlas.cluster.name' in Atlas Configuration File is used." +
+                    " Cluster name mappings can be configured by user defined properties." +
+                    " See additional detail for detail.")
+            .required(false)
+            .expressionLanguageSupported(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .build();
+
+    static final PropertyDescriptor ATLAS_CONF_CREATE = new PropertyDescriptor.Builder()
+            .name("atlas-conf-create")
+            .displayName("Create Atlas Configuration File")
+            .description("If enabled, 'atlas-application.properties' file will be created in 'Atlas Configuration Directory'" +
+                    " automatically when this processor starts." +
```

Comment:
I think this was meant to read 'when this Reporting Task starts', rather than 'when this processor starts'

Reply:
Thank you for finding it!